tests/extmod/socket_fileno.py

```diff
@@ -0,0 +1,17 @@
+# Test socket.fileno() functionality
+
+try:
+    import socket, errno
```

Comment:
```suggestion
    import socket
```

`errno` isn't being used